docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,502 @@
 
-* Since the second node has a greater term it has a fresher cluster state, so
-  it is better to unsafely bootstrap the cluster using this node:
+* Run ``crate-node unsafe-bootstrap`` on the second remaining node and answer
+  ``y`` at the confirmation step.
 
 .. code-block:: console
 
     node_2$ ./bin/crate-node unsafe-bootstrap
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
     Current node cluster state (term, version) pair is (5, 3)
 
-    You should only run this tool if you have permanently lost half or more
-    of the master-eligible nodes in this cluster, and you cannot restore the
-    cluster from a snapshot. This tool can cause arbitrary data loss and its
-    use should be your last resort. If you have multiple surviving master
-    eligible nodes, you should run this tool on the node with the highest
-    cluster state (term, version) pair.
-
     Do you want to proceed?
 
-    Confirm [y/N] y
-    Master node was successfully bootstrapped
+    Confirm [y/n] y
+    Master node was successfully bootstrapped.
 
-Detaching nodes from their cluster
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Detaching nodes from the failed cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After unsafely bootstrapping a new cluster, run the ``crate-node detach-cluster``
-command to detach all remaining nodes from the failed cluster so they can join
-the new cluster:
+You can now detach all remaining nodes from the failed cluster so that they can
+join the new cluster.
 
-.. code-block:: console
+You should only run this tool if you have permanently lost all of the
+master-eligible nodes in this cluster. Also only run this tool if you cannot
+restore the cluster from a snapshot and you have already unsafely bootstrapped a
+new cluster by running ``crate-node unsafe-bootstrap`` on a master-eligible node
+that belonged to the same cluster as this node.
 
-    node_3$ ./bin/crate-node detach-cluster
+.. NOTE: Before you continue with the next step, you need to stop CrateDB.
 
-        WARNING: CrateDB MUST be stopped before running this tool.
+.. WARNING:: This tool can cause arbitrary data loss, and its use should be your
+             last resort.
 
-    You should only run this tool if you have permanently lost all of the
-    master-eligible nodes in this cluster and you cannot restore the cluster
-    from a snapshot, or you have already unsafely bootstrapped a new cluster
-    by running ``crate-node unsafe-bootstrap`` on a master-eligible
-    node that belonged to the same cluster as this node. This tool can cause
-    arbitrary data loss and its use should be your last resort.
+Run ``crate-node detach-cluster`` and answer ``y`` at the confirmation step.
+
+.. code-block:: console
+
+    node_3$ ./bin/crate-node detach-cluster
 
     Do you want to proceed?
 
-    Confirm [y/N] y
-    Node was successfully detached from the cluster
\ No newline at end of file
+    Confirm [y/n] y
```

Comment:
N is uppercase on purpose, since it is the default option when you just press enter. This is common with command line tools. So it should be Confirm [y/N]